examples/plot-cap-extSL.jl

```diff
@@ -1,13 +1,18 @@
 module Script
 using CarboKitten.Visualization
 using GLMakie
+import OpenScienceFramework as OSF
+
 
 function main()
     f = Figure()
     plot_crosssection(f[1, 1], "data/ca-extSL.h5")
-    save("docs/src/fig/b13-crosssection-ext.png", f)
+    save("C:/Users/Liu00141/OneDrive - Universiteit Utrecht/Desktop/work_by_week/external plots/b13-crosssection-ext.png", f)
```

Comment:
You can't use an absolute path on your computer

Reply:
my bad here